pkgs/applications/emulators/86box/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, makeWrapper, freetype, SDL2
+, glib, pcre2, openal, rtmidi, fluidsynth, jack2, alsa-lib, qt5, libvncserver
+, discord-gamesdk, libpcap
+
+, enableDynarec ? with stdenv; isx86_32 || isx86_64 || isAarch64 || isAarch32
+, enableNewDynarec ? with stdenv; enableDynarec && (isAarch64 || isAarch32)
+, enableFluidsynth ? true
+, enableRtMidi ? true
+, enableVncRenderer ? false
+, enableMinitrace ? false
+, enableGdbStub ? false
+, unfreeEnableDiscord ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "86Box";
+  version = "3.11";
+
+  src = fetchFromGitHub {
+    owner = "86Box";
+    repo = "86Box";
+    rev = "v${version}";
+    hash = "sha256-n3Q/NUiaC6/EZyBUn6jUomnQCqr8tvYKPI5JrRRFScI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeWrapper
+    qt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    freetype
+    SDL2
+    glib
+    openal
+    pcre2
+    jack2
+    libpcap
+    qt5.qtbase
+    qt5.qttools
+  ] ++ lib.optional (stdenv.isLinux) alsa-lib
+    ++ lib.optional (enableRtMidi) rtmidi
+    ++ lib.optional (enableVncRenderer) libvncserver;
```

Comment:
```suggestion
  ] ++ lib.optional stdenv.isLinux alsa-lib
    ++ lib.optional enableRtMidi rtmidi
    ++ lib.optional enableVncRenderer libvncserver;
```
those are optional

Reply:
OK, I've removed superfluous parens in my lib.optional calls.